utils/bazel/llvm-project-overlay/mlir/BUILD.bazel

```diff
@@ -7851,7 +7887,16 @@ cc_library(
     srcs = glob([
         "lib/Transforms/*.cpp",
     ]),
-    hdrs = glob(["include/mlir/Transforms/*.h"]),
+    hdrs = [
```

Comment:
(likewise, it'd be nice if this could remain as `glob(["include/mlir/Transforms/*.h"])`)